### PR TITLE
Update: include qa-mode to prepare folder for qa-tool

### PR DESCRIPTION
**Pull request by MarciaAr**

### CODE REVIEW

File: optimize.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/python
 
+from __future__ import barry_as_FLUFL
```
Comment (mlimper) [inline optimize.py]: is this really necessary? ;)

File: optimize.py [hunk 2]
```diff
@@ -28,7 +33,7 @@
 parser.add_argument("-t", "--target",              help="target parameter to be used for RapidCompact -c command (example: 1MB, default: use decimation target from config file)", default="")
 parser.add_argument("-s", "--suffix",              help="suffix to be used for output file name", default="_web")
 parser.add_argument("-d", "--delete_output_first", help="if specified, content of the output directory will be deleted (cleaned up) before processing", action="store_true")
-parser.add_argument("-r", "--rapidcompact_exe",    help="RapidCompact CLI executable", default="rpdx")
```
Comment (mlimper) [inline optimize.py:31]: it looks like this argument got lost somehow - let's keep the -r parameter

File: optimize.py [hunk 3]
```diff
@@ -39,9 +44,8 @@
 configFile      = argsDict["configFile"]
 compactTarget   = argsDict["target"]
 outputSuffix    = argsDict["suffix"]
-rpdxExe         = argsDict["rapidcompact_exe"]
```
Comment (mlimper) [inline optimize.py:42]: see comment above

**Comment by mlimper** [comment]
Thanks! : )

Added some comments (see above)

Would you mind addressing them and in addition removing the changes (you can use a .gitignore file if you like) for the "output" directory? They seem minor:
image

**Comment by mlimper** [comment]
(to do so, you can just update your main branch, this PR should then be updated automatically)

**Comment by MarciaAr** [comment]
I updated my branch. And actually I have a .gitignore file with "output", I don't know why it still wants to update.

**Pull request closed without merging**